versioning.asciidoc

```diff
@@ -0,0 +1,20 @@
+= Versioning Gitenberg books
+
+Gitenberg uses http://semver.org/[Semantic Versioning] to describe the changes to books over time.
+The current version of a GITenberg book is stored in two places:
+
```

Comment:
0) Asciidoctor uses this value to produce some kind of version metadata (if not now, it should)
1) in case the asciidoc gets separated from the git repo

Regardless, you are correct.  The version should be expressed in the metadata. aThe `gitberg` tool should update the asciidoc file and the git tag based on the contents of the metadata file.  It should also manage updating the asciidoc file.